types/graphql/language/ast.d.ts

```diff
@@ -431,27 +505,82 @@ export interface EnumTypeDefinitionNode {
 export interface EnumValueDefinitionNode {
     kind: 'EnumValueDefinition';
     loc?: Location;
+    description?: StringValueNode;
     name: NameNode;
     directives?: DirectiveNode[];
 }
 
 export interface InputObjectTypeDefinitionNode {
     kind: 'InputObjectTypeDefinition';
     loc?: Location;
+    description?: StringValueNode;
     name: NameNode;
     directives?: DirectiveNode[];
     fields: InputValueDefinitionNode[];
 }
 
-export interface TypeExtensionDefinitionNode {
-    kind: 'TypeExtensionDefinition';
-    loc?: Location;
-    definition: ObjectTypeDefinitionNode;
-}
+export type TypeExtensionNode =
+  | ScalarTypeExtensionNode
```

Comment:
@dyst5422 Great PR 👍 

One problem I see is the inconsistency in indent (4vs2). But I think it should be handled in separate PR. Ideally, someone would copy [.prettierrc](https://github.com/graphql/graphql-js/blob/master/.prettierrc) from `graphql-js` and run it on entire `types/graphql/` folder.

Reply:
Fair enough. I didn't want to run prettier since it would use my own settings.